patches/parent-origin.diff

```diff
@@ -0,0 +1,25 @@
+Remove parentOriginHash checko
+
+This fixes webviews from not working properly due to a change upstream.
+Upstream added a check to ensure parent authority is encoded into the webview
+origin. Since our webview origin is the parent authority, we can bypass this
+check.
+
+Index: code-server/lib/vscode/src/vs/workbench/contrib/webview/browser/pre/main.js
+===================================================================
+--- code-server.orig/lib/vscode/src/vs/workbench/contrib/webview/browser/pre/main.js
++++ code-server/lib/vscode/src/vs/workbench/contrib/webview/browser/pre/main.js
+@@ -339,12 +339,7 @@ const hostMessaging = new class HostMess
+ 			throw err instanceof Error ? err : new Error(String(err));
+ 		}
+ 
+-		if (hostname === parentOriginHash || hostname.startsWith(parentOriginHash + '.')) {
+-			// validation succeeded!
+-			return start(parentOrigin);
+-		}
+-
+-		throw new Error(`Expected '${parentOriginHash}' as hostname or subdomain!`);
```

Comment:
I was thinking about how we might generalize this so it could potentially be upstreamed one day and I think what we could do is before doing the whole hash thing (around line 321) we could check to see if the webview host is the same as the parent origin and if so we can bypass the check since we know it is safe.

So maybe something like:

```
// It is safe to run if we are on the same domain.
const parent = new URL(parentOrigin)
if (parent.hostname == location.hostname) {
  return start(parentOrigin)
}
```